airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.module.scss

```diff
@@ -18,6 +20,10 @@
   padding: 400px 30px 30px 30px;
 }
 
+.container > *:last-child {
+  padding-bottom: 150px;
```

Comment:
Because this is duplicate here, maybe it should be a scss variable to make sure it's consistent both here and in the MainPageWithScroll.

Reply:
Also was thinking about that... Ok, will move it to a separate variable 👍